docs/README.md

```diff
@@ -0,0 +1,76 @@
+# GitOps for Application Delivery
+
+## Why
+
+As a software organization, I would like to:
+
+* Audit all changes made to pipelines, infrastructure, application make-up
+* Roll forward/back to desired state in case of issues
+* Consistently configure all environments
+* Reduce manual effort by automating application and environment setup, remediation
+* Have an easy way to manage application and infrastructure state across clusters/environments
+
+## What
+
+* GitOps is a natural evolution of Agile and DevOps methodologies.
+* GitOps is when the infrastructure and/or application state is fully represented by the contents of a git repository. Any changes to the git repository are reflected in the corresponding state of the associated infrastructure and applications through automation.
+
+## Principles
+
+* Git as the source of truth.
+* Separate application source code (Java/Go) from deployment manifests i.e the application source code and the GitOps configuration reside in separate git repositories.
+* Deployment manifests are standard Kubernetes (k8s) manifests i.e Kubernetes manifests in the GitOps repository can be simply applied with nothing more than a `oc apply`.
+* Kustomize for defining the differences between environments i.e reusable parameters with extra resources described using kustomization.yaml
+
+## Getting Started
+#### [Gitops version: v0.0.2](https://github.com/redhat-developer/gitops-cli/releases/tag/v0.0.2)
+### Steps
+
+The following steps will guide you to setup GitOps for application delivery:
+
+1. [GitOps Day 1 Operations](./journey/day1): Install the prerequisites and setup your GitOps Pipeline.
+2. [GitOps Day 2 Operations](./journey/day2): Continue adding more applications.
+
+### Reference
+
+The command reference to generate and manage the above can be found [here](./commands).
+
+This project provides the tooling to generate the recommended directories and manifests as per the [Application Pipelines Model](./model).
+
+### Sample
+
+A sample GitOps repository can be found [here](https://github.com/rhd-gitops-example/gitops).
```

Comment:
Should we update this sample repo?  It is 2 months old.

Reply:
Yes William !! can create a new task to make changes to this repo ? should we also move that to red hat developer org? @wtam2018

Reply:
Good point.   I copied the old sample gitops repo to https://github.com/redhat-developer/gitops-repo-example.  Please update all references to point to the new example gitops repo location.   

You can update the content of the new example repo later and use this https://issues.redhat.com/browse/GITOPS-290.   Don't need to create a new Jira.   Thanks.

Reply:
You should scan the document.   There should be any references to gitops-repo-example anymore.